docs/manual/common/guide/devmode/DevEnvironment.md

```diff
@@ -83,14 +83,16 @@ This all happens automatically without special code or additional configuration.
 
 <!--end copied section -->
 
-You can verify that your services are running by viewing `http://localhost:8000/services` in a web browser (or with a command line tool such as `curl`).  The Service Locator, running on port 8000, will return JSON such as:
+You can verify that your services are running by viewing `http://localhost:9008/services` in a web browser (or with a command line tool such as `curl`). The Service Locator, running on port `9008`, will return JSON such as:
 
 ```
 [{"name":"hello-stream","url":"http://0.0.0.0:26230"},
  {"name":"cas_native","url":"tcp://127.0.0.1:4000/cas_native"},
  {"name":"hello","url":"http://0.0.0.0:24266"}]
 ```
 
+Recently the port for service locator was changed from `8000` to `9008`.
+
```

Comment:
Maybe this is not necessary. 
This change is rather transparent for the users and we can better announce it on the release notes. 

It may happen that someone has a script in dev environment that will stop to work, but that can be easily detected as the port is printed out on the console during bootstrap. The user will then quickly discover this change.

Reply:
ack, I'll remove those notes